.travis.yml

```diff
@@ -0,0 +1,135 @@
+      script:
+        - pytest
+
+      # MacOS (OSX) Jobs
+    - os: osx
+      language: sh
+      python: 3.6
+      env: PYVER="3.6.10"
+      install:
+        - pip3 install --upgrade pip
+        - pip3 install -r ./.travis/travis-requirements.txt
+        - pip3 install pycairo
+        - pip3 install .
+      script:
+        - pytest
+
+    - os: osx
+      language: sh
+      python: 3.7
+      env: PYVER="3.7.7"
+      install:
+        - pip3 install --upgrade pip
+        - pip3 install -r ./.travis/travis-requirements.txt
+        - pip3 install pycairo
+        - pip3 install .
+      script:
+        - pytest
+
+    - os: osx
+      language: sh
+      python: 3.8
+      env: PYVER="3.8.0"
+      install:
+        - pip3 install --upgrade pip
+        - pip3 install -r ./.travis/travis-requirements.txt
+        - pip3 install pycairo
+        - pip3 install .
+      script:
+        - pytest
+
+      # Windows Jobs
+    - os: windows
+      language: sh
+      python: 3.6
+      # the latest version available on chocolatey is 3.6.8
+      env: PYVER="3.6.8" PYDIR="Python36"
+      install:
+        - choco install python --version=$PYVER
+        - export PATH="/c/$PYDIR:/c/$PYDIR/Scripts:$PATH"
+        - choco install ffmpeg
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --upgrade pip
+        - python -m pip install --user -r ./.travis/travis-requirements.txt
+        - python ./.travis/cairo-windows.py
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --user .
+      script: 
+        - python -m pytest
+
+    - os: windows
+      language: sh
+      python: 3.7
+      env: PYVER="3.7.7" PYDIR="Python37"
+      install:
+        - choco install python --version=$PYVER
+        - export PATH="/c/$PYDIR:/c/$PYDIR/Scripts:$PATH"
+        - choco install ffmpeg
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --upgrade pip
+        - python -m pip install --user -r ./.travis/travis-requirements.txt
+        - python ./.travis/cairo-windows.py
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --user .
+      script: 
+        - python -m pytest
+
+    - os: windows
+      language: sh
+      python: 3.8
+      env: PYVER="3.8.0" PYDIR="Python38"
+      install:
+        - choco install python --version=$PYVER
+        - export PATH="/c/$PYDIR:/c/$PYDIR/Scripts:$PATH"
+        - choco install ffmpeg
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --upgrade pip
+        - python -m pip install --user -r ./.travis/travis-requirements.txt
+        - python ./.travis/cairo-windows.py
+        - cmd.exe //c "RefreshEnv.cmd"
+        - python -m pip install --user .
+      script: 
+        - python -m pytest
+
+before_install:
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then chmod +x ./.travis/osx.sh; sh ./.travis/osx.sh; fi
+
+branches:
+  only:
+  - travis-setup
+
+notifications:
+  email: false
```

Comment:
please add a newline at the end of the file :)

Reply:
```suggestion
  email: false

```